NuGet.config

```diff
@@ -12,6 +12,7 @@
     <add key="dotnet8-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8-transport/nuget/v3/index.json" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
+    <add key="dotnet-libraries-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries-transport/nuget/v3/index.json" />
```

Comment:
Why do we have so many NuGet feeds?

Reply:
My guess is that these aren't all needed. I would bet that dotnet6* and dotnet7* can be eliminated. Maybe nuget-build too.

Reply:
https://github.com/dotnet/roslyn/pull/69928